### DISCUSSION TITLE
Unable to Create Permissions, Roles, and Edit Themes

### DISCUSSION THREAD

**Discussion by bingyulee** [fief-dev/fief#236] [General]
I am self-hosting using the docker image ghcr.io/fief-dev/fief:latest. The server can be activated normally, but I found that I cannot click on the settings for access control and customization. I tried to add permissions directly when editing a user but was unsuccessful. When I check the browser's console, a 400 bad request error appears. If I directly open the link /admin/permissions, a 404 not found error occurs. Could anyone suggest what might be causing this? Thank you.

**Comment by fief-bailiff** [comment]
Hail, @bingyulee 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by bingyulee** [comment]
I have found what the problem is, the sidebar's access control cannot unfold to display the Permissions and Roles settings when I click on it. If I directly open /admin/access-control/permissions, I can open the settings page. I'm not sure if this is a bug?

**Reply by frankie567** [reply]
Hey @bingyulee 👋  You probably hit issue #54. Would you be able to try with a bigger window/screen?

**Reply by bingyulee** [reply]
Yes, you're correct. It appears that the screen needs to be set to over 1535px to display correctly. I used the device simulation function of the browser and set the width to over 1535px. In this case, the options include both icons and link text, and only then can the submenu be displayed when clicked. Once the width is reduced to under 1535, when it turns into a single icon, the submenu cannot be opened. Additionally, my screen display is 2560x1600, I wonder why it cannot display properly?
